extensions/replication/tasks/MultipleBackendTask.js

```diff
@@ -144,16 +144,19 @@ class MultipleBackendTask extends ReplicateObject {
             range,
         });
         const doneOnce = jsutil.once(done);
-        const sourceReq = this.backbeatSource.getObject({
-            Bucket: sourceEntry.getBucket(),
-            Key: sourceEntry.getObjectKey(),
-            VersionId: sourceEntry.getEncodedVersionId(),
-            // A 0-byte part has no range.
-            Range: range && `bytes=${range.start}-${range.end}`,
-            LocationConstraint: sourceEntry.getDataStoreName(),
-        });
         // A 0-byte object has no range, otherwise range is inclusive.
         const size = range ? range.end - range.start + 1 : 0;
+        let sourceReq = null;
```

Comment:
`sourceReq` will need to be a request obj or there will be problems in `_putMPUPart`. I'm not too familiar how the request looks here, but I'm thinking one possible way is to build a request object with relevant data since we have the `sourceEntry` here

Reply:
thanks! will fix this asap

Reply:
Changes have been updated and have been tested on a remote deployment. 
I have tested 0 byte parts for MPUs and replication works :ok_hand: 
If anyone would like to sit down to double check the tests with me, please lmk.

Reply:
Great! I'd be sure to also run the Zenko CRR end-to-end test suite.

Reply:
On it, thanks for the reminder

Reply:
How did you test 0 byte parts if the minimum part size is 5mb?

Reply:
You can commit MPUs where the last part is 0-byte
or if there is only 1 part, and it's 0-byte